ntpclient/ntpclient.go

```diff
@@ -30,30 +29,36 @@ type ntppacket struct {
 // NTPClient create's a new NTPClient and returns local based on ntp servers provided timestamp
 func NTPClient(pool []string) (time.Time, error) {
```

Comment:
Since we don't use err, we could do a check at the top to check if len(pool) == 0 and complain if it's empty

Reply:
I will remove the error return value 
a pool should never be empty because of https://github.com/thrasher-corp/gocryptotrader/blob/master/config/config.go#L1250